app/src/main/java/com/golfbetterapp/golfbetter/clients/PgaAppEngineClient.java

```diff
@@ -35,145 +33,162 @@
  */
 public class PgaAppEngineClient {
   public static final String TOURNAMENT_FORMAT = "{tournament}";
-  public static final String TOURNAMENTS_ENDPOINT = "https://stoked-depth-428423-j7.uc.r.appspot.com/api/v1/pga-tournaments";
-  public static final String LEADERBOARD_PLAYERS_ENDPOINT = "https://stoked-depth-428423-j7.uc.r.appspot.com/api/v1/pga-leaderboard-players?tournamentId=" + TOURNAMENT_FORMAT;
-  public static final String PLAYER_SCORECARD_ENDPOINT = "https://stoked-depth-428423-j7.uc.r.appspot.com/api/v1/pga-player-scorecards?tournamentId=" + TOURNAMENT_FORMAT;
+
+  public static final String APP_SERVER_URL = "https://stoked-depth-428423-j7.uc.r.appspot.com";
+  // public static final String APP_SERVER_URL = "http://10.0.2.2:8080"; // For local dev
```

Comment:
we should make this easier and configurable at some point, but for now if you want your emulator to communicate with a local version of app-engine running, you need to switch these values